apps/src/javalab/Neighborhood.js

```diff
@@ -34,5 +35,13 @@ export default class Neighborhood {
     controller.subtype.createDrawer(svg);
     controller.subtype.initWallMap();
     controller.initWithSvg(svg);
+
+    const slider = document.getElementById('slider');
+    this.speedSlider = new Slider(10, 35, 130, slider);
+    // TODO: use the speed slider to adjust the pegman speed
```

Comment:
Not required for this change, but it's a bonus if you do: 
You have all the knowledge for how to calculate the pegman speed. Adding a function called "getPegmanSpeed" or similar will let us use that knowledge as soon as we start implementing the maze API calls.